migrate/20250130_kubernetes_vm_size.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+Sequel.migration do
+  change do
+    alter_table(:kubernetes_nodepool) do
+      add_column(:target_node_size, :text, collate: '"C"', null: false, default: "standard-2")
```

Comment:
Other models set the default value through clover instead of at the database level. Should we follow that approach and not set the default value in the database?

Reply:
Done!